json.go

```diff
@@ -93,35 +93,41 @@ func Protect(handler http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+// WrapJSONRequestHandlerFunc sets up an HTTP handler for use with JSON APIs with standard logging.
+// http.Requests will have a logger (with a request ID/method/path logged) attached to the Context.
+// This can be accessed via GetLogger(Context).
+func WrapJSONRequestHandlerFunc(w http.ResponseWriter, req *http.Request) {
```

Comment:
I called it `*HandlerFunc` because it has the same signature as an http.HandlerFunc ( https://golang.org/pkg/net/http/#HandlerFunc ) but it sets up the logging.

That said, I agree. Update coming shortly.